src/pages/Login.js

```diff
@@ -1,27 +1,21 @@
-import React ,{useState} from "react";
+import React, { useState } from "react";
 import "../styles/login.css";
```

Comment:
make these two CSS links into one link of CSS, either importing any one of them in one file or making a separate CSS file to import all the CSS links into one file

Reply:
Thanks for the sugestion. I have updated the PR with these changes.